src/Microsoft.AspNet.Mvc.Razor/Precompilation/RazorPreCompiler.cs

```diff
@@ -25,6 +25,8 @@ namespace Microsoft.AspNet.Mvc.Razor.Precompilation
 {
     public class RazorPreCompiler
     {
+        private const string PathSeparator = "/";
```

Comment:
May need a more specific name since this is not about path separation in general, just for the compiler cache e.g. `CacheKeyDirectorySeparator`.